### PR TITLE
debug monster commands

### PR DESCRIPTION
**Pull request by qndel**

### CODE REVIEW

**Comment by julealgon** [comment]
Can you please add some description of what you are adding here, without forcing people to look at the actual changes to see them?

**Comment by qndel** [comment]
> Can you please add some description of what you are adding here, without forcing people to look at the actual changes to see them?

No. The point of PRs is to have other people look at the code xD

File: Source/debug.cpp [hunk 1]
```diff
@@ -502,6 +595,8 @@ std::vector<DebugCmdItem> DebugCmdList = {
 	{ "cursorcoords", "Toggles showing cursor coords.", "", &DebugCmdShowCursorCoords },
 	{ "grid", "Toggles showing grid.", "", &DebugCmdShowGrid },
 	{ "seedinfo", "Show seed infos for current level.", "", &DebugCmdLevelSeed },
+	{ "mon", "Spawns monster {name}.", "{name}", &DebugCmdSpawnMonsterSingle },
+	{ "mmon", "Spawns up to 8 monsters {name}.", "{name}", &DebugCmdSpawnMonsterMany },
```
Comment (AJenbo) [inline Source/debug.cpp:599]: How about making this a single command with an optional paramter, also rename it to spawn:

```suggestion
	{ "spawn", "Spawns monster [count] {name}.", "{name}", &DebugCmdSpawnMonsterSingle },
```

`spawn 3 skeleton`

`spawn scavenger`

Comment (qndel) [reply Source/debug.cpp:599]: if you code that, sure ;)

**Comment by AJenbo** [comment]
Merged via https://github.com/diasurgical/devilutionX/commit/d47d2e3030237c4661c53f89001fe17e4eee4f94

**Pull request closed without merging**